engine/modelc/README.md

```diff
@@ -0,0 +1,59 @@
+
+
+# Model Importer
```

Comment:
I added a readme, so that we will remember that it's a lot easier to debug the model loading now with a C++ backend.